### PR TITLE
DM-28892: BaseCsc: improve --simulate command-line argument help and simulation_mode constructor argument handling

### PR DESCRIPTION
**Pull request by r-owen**

### CODE REVIEW

File: python/lsst/ts/salobj/config_schema.py [hunk 1]
```diff
@@ -0,0 +1,64 @@
+# This file is part of ts_salobj.
+#
+# Developed for the LSST Telescope and Site Systems.
+# This product includes software developed by the LSST Project
```
Comment (wvreeven) [inline python/lsst/ts/salobj/config_schema.py:4]: Please use Vera Rubin Observatory at least in line 3 but preferrably also in line 4.

**Review by wvreeven — APPROVED** [review]
One minor comment but otherwise this looks good to me.

File: python/lsst/ts/salobj/config_schema.py (hunk 1 above)
Comment (r-owen) [reply python/lsst/ts/salobj/config_schema.py:4]: I updated line 3. I'm leaving line 4 for now as I don't know what else to say (and it matches all my other packages).

Comment (wvreeven) [reply python/lsst/ts/salobj/config_schema.py:4]: Great. Yes we do not really have a consensus for line 4 AFAIK.

Comment (r-owen) [reply python/lsst/ts/salobj/config_schema.py:4]: I also filed https://jira.lsstcorp.org/browse/DM-28930 in hopes of getting a more official answer (though likely too late for this ticket)

**Pull request merged**